packages/control-property-editor/src/App.tsx

```diff
@@ -92,9 +93,14 @@ export default function App(appProps: AppProps): ReactElement {
         setWarningDialogVisibility(false);
     }
 
+    function closeAdpWarningDialog(): void {
```

Comment:
It would be nice to use an arrow function handler here
```suggestion
    const closeAdpWarningDialog = (): void => {
```

Reply:
Done